### PR TITLE
Merge 1.4 to master

### PR DESCRIPTION
**Pull request by grena**
| Q                 | A
| ----------------- | ---
| Added Specs       |  Y
| Added Behats      | Y
| Travis CI is ok   | Y
| Changelog updated | Y

### CODE REVIEW

File: features/Context/Page/Job/Show.php [hunk 1]
```diff
@@ -20,11 +20,9 @@ class Show extends BaseIndex
...
     public function getPropertyErrorMessage($property)
     {
-        $error = $this->find('css', sprintf('tr:contains("%s") span.label-important', ucfirst($property)));
-
-        if (!$error) {
-            throw new \InvalidArgumentException(sprintf('Could not find the %s property', $property));
-        }
+        $error = $this->spin(function () use ($property) {
+            return $this->find('css', sprintf('tr:contains("%s") span.label-important', ucfirst($property)));
+        }, 20, sprintf('Could not find the %s property', $property));
```
Comment (skeleton) [inline features/Context/Page/Job/Show.php]: remove "20" timeout

File: features/Context/Page/Base/Form.php [hunk 2]
```diff
@@ -292,12 +292,18 @@ public function openAvailableAttributesMenu()
      */
     public function addAvailableAttributes(array $attributes = [])
     {
-        $this->openAvailableAttributesMenu();
+        $this->spin(function () {
+            return $this->find('css', $this->elements['Available attributes button']['css']);
+        }, 20, sprintf('Cannot find element "%s"', $this->elements['Available attributes button']['css']));
```
Comment (juliensnz) [inline features/Context/Page/Base/Form.php]: need to be reworked, the timeout parameter does not exists anymore

File: app/SymfonyRequirements.php [hunk 3]
```diff
@@ -447,6 +447,12 @@ public function __construct()
         }
 
         $this->addRequirement(
+            function_exists('iconv'),
```
Comment (juliensnz) [inline app/SymfonyRequirements.php]: finally

File: features/Context/Page/Job/Show.php (hunk 1 above)
Comment (juliensnz) [inline features/Context/Page/Job/Show.php]: same here: drop the timeout paremeter :)

**Comment by juliensnz** [comment]
GTM after comment  :)

**Comment by phaseinducer** [comment]
GTM !

**Pull request merged**